lib/node_modules/@stdlib/math/base/special/sincos/lib/index.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+/**
+* Simultaneously compute the sine and cosine of a number.
+*
+* @module @stdlib/math/base/special/sincos
+*
+* @example
+* var sincos = require( '@stdlib/math/base/special/sincos' );
+*
+* var v = sincos( 0.0 );
+* // returns [ ~0.0, ~1.0 ]
+*
+* v = sincos( Math.PI/2.0 );
+* // returns [ ~1.0, ~0.0 ]
+*
+* v = sincos( -Math.PI/6.0 );
+* // returns [ ~-0.5, ~0.866 ]
+*
+* v = sincos( NaN );
+* // returns [ NaN, NaN ]
+*
+* var out = new Float64Array( 2 );
+*
+* var v = sincos( out, 0.0 );
```

Comment:
I've separated this into two examples so that this should no longer be a re-declaration. ✅